system-addon/content-src/message-center/components/SnippetBase.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+const defaultStyles = {
+  container: {
+    position: "fixed",
+    bottom: 0,
+    left: 0,
+    right: 0,
+    padding: "20px",
+    backgroundColor: "white",
+    fontSize: "12px",
+    lineHeight: "16px",
+    boxShadow: "0 -1px 4px 0 rgba(12, 12, 13, 0.1)"
+  },
+  innerWrapper: {
+    maxWidth: "992px",
+    margin: "0 auto",
+    display: "flex",
+    alignItems: "center"
+  },
+  blockButton: {
+    background: "none",
+    border: 0,
+    display: "block",
+    position: "absolute",
+    top: "10px",
+    right: "10px",
+    height: "16px",
+    width: "16px",
+    backgroundImage: "url(resource://activity-stream/data/content/assets/glyph-dismiss-16.svg)",
```

Comment:
not centered
<img width="84" alt="image" src="https://user-images.githubusercontent.com/810040/37911999-bd9a888e-3111-11e8-87f3-f4a31cd81d35.png">